### ISSUE TITLE
[Bug] stats from Finnhub BA doesn't work

### ISSUE THREAD

**Issue by JerBouma** [GamestonkTerminal/GamestonkTerminal#1384]
**Describe the bug**
No data available, does this make sense?

```
2022 Feb 16, 10:47 (✨) /stocks/ba/ $ load tsla

Loading Daily TSLA stock with starting period 2019-02-11 for analysis.
s
Datetime: 2022 Feb 16 10:47
Timezone: America/New_York
Currency: USD
Market:   CLOSED

2022 Feb 16, 10:47 (✨) /stocks/ba/ $ stats
No sentiment stats found.

2022 Feb 16, 10:48 (✨) /stocks/ba/ $ load amzn

Loading Daily AMZN stock with starting period 2019-02-11 for analysis.
s
Datetime: 2022 Feb 16 10:48
Timezone: America/New_York
Currency: USD
Market:   CLOSED

2022 Feb 16, 10:48 (✨) /stocks/ba/ $ stats
No sentiment stats found.

2022 Feb 16, 10:48 (✨) /stocks/ba/ $ load aapl

Loading Daily AAPL stock with starting period 2019-02-11 for analysis.

Datetime: 2022 Feb 16 10:48
Timezone: America/New_York
Currency: USD
Market:   CLOSED

2022 Feb 16, 10:48 (✨) /stocks/ba/ $ stats
No sentiment stats found.

2022 Feb 16, 10:48 (✨) /stocks/ba/ $ load googl

Loading Daily GOOGL stock with starting period 2019-02-11 for analysis.

Datetime: 2022 Feb 16 10:48
Timezone: America/New_York
Currency: USD
Market:   CLOSED

2022 Feb 16, 10:48 (✨) /stocks/ba/ $ stats
No sentiment stats found.
```

**Comment by minhhoang1023** [comment]
@JerBouma This is a Premium feature so it'd require submission. However, in my opinion, it should be clear which one is Premium feat and which is not. Wdyt?

**Comment by JerBouma** [comment]
@minhhoang1023 Hmm, can't we capture the error message here? I presume it would say something like "Premium Feature.. please subscribe"

**Comment by minhhoang1023** [comment]
@JerBouma The error message is simply `{"error":"You don't have access to this resource."}`

**Comment by minhhoang1023** [comment]
Update: this will be addressed with #1357

**Comment by minhhoang1023** [comment]
close as we no longer have `stats`, and only have `news` from this PR #1421
